### PR TITLE
test commit

### PR DESCRIPTION
**Pull request by OlhaVarenyk**

### CODE REVIEW

File: playwright.config.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : undefined,
+  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+  reporter: 'html',
+  /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+  use: {
+    /* Base URL to use in actions like `await page.goto('/')`. */
+    // baseURL: 'http://127.0.0.1:3000',
```
Comment (hlazkov) [inline playwright.config.js]: Its better to have baseUrl set here, not in tests or anywhere.

File: tests/myTest.spec.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  await expect(page.locator('[data-test="shopping-cart-link"]')).toBeVisible();
+  expect(await page.locator('[data-test="inventory-item"]').count()).toBeGreaterThan(1);
+});
+
+test("test2", async ({ page }) => {
+  await page.goto("https://www.saucedemo.com/");
+  await page.locator('[data-test="username"]').fill("standard_user");
+  await page.locator('[data-test="password"]').fill("secret_sauce");
+  await page.locator('[data-test="login-button"]').click();
+  const productName = await page.textContent('[data-test="inventory-item-name"]');
```
Comment (hlazkov) [inline tests/myTest.spec.js]: This locator will resolve to all product names. I would suggest to pick first.
Also Its better to work with page with approach: `await page.locator(someLocator).textContent();` or `await page.getByTestId(someTestId).textContent();`. With such approach you will have page.locator.interaction, this code is better in sence of support and maintain.

File: tests/myTest.spec.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  await page.locator('[data-test="add-to-cart-sauce-labs-backpack"]').click();
+  await expect(page.locator('[data-test="shopping-cart-badge"]')).toContainText("1");
+  await page.locator('[data-test="shopping-cart-link"]').click();
+  expect(await page.textContent('[data-test="item-4-title-link"]')).toBe(productName);
```
Comment (hlazkov) [inline tests/myTest.spec.js]: Like for await in the correct place :)

File: tests/myTest.spec.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  await page.locator('[data-test="remove-sauce-labs-backpack"]').click();
+  await page.locator('[data-test="inventory-item"]').isHidden();
```
Comment (hlazkov) [inline tests/myTest.spec.js]: Should we have expect here instead of calling `.isHidden`?

**Review by hlazkov — CHANGES_REQUESTED** [review]
Just a few comments to fix. Good job here!

File: tests/myTest.spec.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import { test, expect } from "@playwright/test";
+test.describe("Testing", () => {
+  test("test1", async ({ page }) => {
```
Comment (AlinaZhykharieva) [inline tests/myTest.spec.js]: names: test1 and test2 say nothing to us in the report. I would suggest to add more meaningful names for the scenarioas

**Review by AlinaZhykharieva — APPROVED** [review]

**Comment by AlinaZhykharieva** [comment]
Я б ще додала інформації до README, як запускати, що треба мати

**Pull request merged**